R/derive_param_computed.R

```diff
@@ -251,7 +304,15 @@
 #'   constant_by_vars = exprs(USUBJID)
 #' )
 #'
-#' # Example 3: Using data from an additional dataset and other variables than AVAL
+#' @caption Example 3 - Using `dataset_add` argument and non-`AVAL` variables
+#'
+#' @info Using data from an additional dataset and other variables than `AVAL`
```

Comment:
Can you give a bit more explanation here of what is being used from `dataset_add`?